### PR TITLE
refactor: Boostrap to AntD - Row/Col

### PR DESCRIPTION
**Pull request by michael-s-molina**
### SUMMARY
- Migrates `Row` and `Col` components from Bootstrap to AntD
- Improves `VizTypeControl` layout when resizing

See: #10254

@rusackas @junlincc @pkdotson

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

https://user-images.githubusercontent.com/70410625/114556184-d62b4f00-9c3e-11eb-9478-37eb62e84302.mov

https://user-images.githubusercontent.com/70410625/114556203-d9bed600-9c3e-11eb-868f-e66ab813bdd3.mov

https://user-images.githubusercontent.com/70410625/114556210-dcb9c680-9c3e-11eb-98d9-32113c69f97e.mov

https://user-images.githubusercontent.com/70410625/114556216-de838a00-9c3e-11eb-8c1f-e1bf19fd4121.mov

https://user-images.githubusercontent.com/70410625/114556295-f1965a00-9c3e-11eb-9fc6-b7a076edd0a1.mov

https://user-images.githubusercontent.com/70410625/114556526-2d312400-9c3f-11eb-9fde-cb1a816147d7.mov

https://user-images.githubusercontent.com/70410625/114556661-4e921000-9c3f-11eb-8c19-96934c0dcea8.mov

https://user-images.githubusercontent.com/70410625/114556675-50f46a00-9c3f-11eb-87ba-0143e50eb21d.mov

https://user-images.githubusercontent.com/70410625/114556751-649fd080-9c3f-11eb-902c-31be5169d79f.mov

https://user-images.githubusercontent.com/70410625/114557467-150dd480-9c40-11eb-9f69-f8141abae629.mov

https://user-images.githubusercontent.com/70410625/114557265-e09a1880-9c3f-11eb-8249-26570aaa4967.mov

https://user-images.githubusercontent.com/70410625/114557278-e3950900-9c3f-11eb-8bc8-9bc6d9afe2a7.mov

https://user-images.githubusercontent.com/70410625/114557327-f27bbb80-9c3f-11eb-9c1f-108890f84d95.mov

https://user-images.githubusercontent.com/70410625/114557342-f576ac00-9c3f-11eb-84b1-07d730304beb.mov

https://user-images.githubusercontent.com/70410625/114557368-fad3f680-9c3f-11eb-8eb5-b6764f6d7f3c.mov

https://user-images.githubusercontent.com/70410625/114557388-fdcee700-9c3f-11eb-8323-9683477a2de4.mov

### TEST PLAN
1 - I added before and after videos for each touched component
2 - Review all touched components

### ADDITIONAL INFORMATION
- [x] Has associated issue:
- [x] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by pkdotson — APPROVED** [review]
Pulled the code and everything looks good to me!

File: superset-frontend/src/SqlLab/components/SaveQuery.tsx [hunk 1]
```diff
@@ -114,7 +115,7 @@ export default function SaveQuery({
   const renderModalBody = () => (
     <FormGroup bsSize="small">
       <Row>
-        <Col md={12}>
+        <Col xs={24}>
```
Comment (rusackas) [inline superset-frontend/src/SqlLab/components/SaveQuery.tsx:118]: This seems OK, but if there's only one `Col` for each of these rows, I'm wondering if we even need the rows/cols at all. It seems like a good ol' `div` would suffice.

File: superset-frontend/src/SqlLab/components/ScheduleQueryButton.tsx [hunk 2]
```diff
@@ -179,7 +180,7 @@ const ScheduleQueryButton: FunctionComponent<ScheduleQueryButtonProps> = ({
       </Row>
       {scheduleQueryWarning && (
         <Row>
-          <Col md={12}>
+          <Col xs={24}>
```
Comment (rusackas) [inline superset-frontend/src/SqlLab/components/ScheduleQueryButton.tsx:183]: Not sure we even need these either. Not saying it should affect this PR, just something to consider.

File: superset-frontend/src/explore/components/controls/BoundsControl.jsx [hunk 3]
```diff
@@ -87,8 +88,8 @@ export default class BoundsControl extends React.Component {
       <div>
         <ControlHeader {...this.props} />
         <FormGroup bsSize="small">
-          <Row>
-            <Col xs={6}>
+          <Row gutter={16}>
```
Comment (rusackas) [inline superset-frontend/src/explore/components/controls/BoundsControl.jsx:91]: I think we could use withTheme for these to get a hold of gridUnit, but it's not a blocker for this work I suppose.

**Review by rusackas — APPROVED** [review]
LGTM, too. Noted a few things we might want to touch up, but I don't think they should prevent this from being merged.

**Pull request merged**